src/neuroconv/datainterfaces/icephys/baseicephysinterface.py

```diff
@@ -112,6 +112,7 @@ def run_conversion(
             Type of icephys recording.
         skip_electrodes : tuple, optional
             Electrode IDs to skip. Defaults to ().
+        save_path: string, optional
```

Comment:
Well, that one's supposed to have been removed a while back...

Can always do it in a follow up